packages/astro/src/core/encryption.ts

```diff
@@ -0,0 +1,65 @@
+import { base64, decodeHex, encodeHex } from '@oslojs/encoding';
+
+const ALGORITHM = 'AES-GCM';
+
+export async function createKey() {
+	const key = await crypto.subtle.generateKey(
+		{
+			name: ALGORITHM,
+			length: 256,
+		},
+		true,
+		['encrypt', 'decrypt']
+	);
+	return key;
+}
+
+export async function importKey(bytes: Uint8Array) {
```

Comment:
The documentation is missing. Also, the function's name suggests that it imports the key from somewhere, so we should explain that part, too. We should explain also what are the `bytes`, and what they are meant for